### DISCUSSION TITLE
Duration of AUDIO

### DISCUSSION THREAD

**Discussion by khabibulloevm** [SYSTRAN/faster-whisper#606] [Q&A]
Is there any limitation regarding the duration of an audio file? My audio is almost 39 minutes long, when I run the model for transcription, when 589 seconds are reached, the model does not proceed further ... At the same time, I don’t get any errors regarding memory or computing resources... @guillaumekln please help me

**Comment by Pengymd** [comment]
i got the same problem.  If i put an audio of 3 minutes, its fine and looks faster then whisper.  When i put a 10 minute audio, my python code just freeze.
